apps/ledger-live-desktop/src/newArch/Collectibles/types/DetailDrawer.ts

```diff
@@ -5,6 +5,8 @@ import { CollectibleType } from "./Collectibles";
 export enum FieldStatus {
   Loading = "loading",
   Loaded = "loaded",
+  Error = "error",
+  NoData = "nodata",
 }
```

Comment:
Maybe rework the function that use that to use TanstackQuery?

Reply:
I think it would need a deeper rework since the current implementation is using a react context, cache, and api call